### ISSUE TITLE
Feature request - Add possibility to sort

### ISSUE THREAD

**Issue by Asiberus** [DomDew/strapi-plugin-fuzzy-search#171]
Hello, 

Is it possible to add the sort option for the REST endpoint, as for the strapi api endpoint ?

Thank you!

**Comment by hyped0001** [comment]
Bump

**Comment by domcoleman** [comment]
Hey @DomDew,
I think Absiberus is asking for a way to perform a fuzzy search but then sorting on a particular field. Currently the only sort option would be relevance, but for example; if people were searching through a collection of products, they may want to sort by price.
Obviously you could use the API route for those products and use `containsi`, but that wouldn't be as beneficial as a fuzzy search would, as it could miss products.

**Comment by Asiberus** [comment]
Yes it's that. In my usecase I provide the user a search on the collection and display the results in a table that can be sort through each column. The search is global among the properties of a item, which does not work with `containsi` that search only in a property (as far as I understood it). 
Curently I use the param `_q`, it is not expose on the API but it works. It is the param used internally by strapi in the admin panel. But it does not work with date or nested params.

The best for me would be to have a query param `sort` that override the sorting provided by the fussysort algorithm.

**Comment by DomDew** [comment]
Ah, okay I see your point now, though I must admit that this comes at a pretty low priority for me at the moment 😅. Feel free to open a PR though if this is an important feature for you and don't hesitate to get back to me if you have any questions!
